.github/workflows/promql-to-dd-go.yaml

```diff
@@ -0,0 +1,16 @@
+name: Test, build and publish
+
+on:
+  push:
+    branches:
+    tags:
+      - v*
+  # Run tests for any PRs.
```

Comment:
I think there's a way to only run certain actions when certain code paths change? it'd be good to not have to run these actions everytime anything in this whole repo changes yeah?

Reply:
Goot pickup @TimSimmons. I was thinking about adding it after a few round of tests but completely forgotten. It's in place now. Please have another look. Thanks.